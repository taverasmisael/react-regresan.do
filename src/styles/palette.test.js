import Palette from './palette'

describe('Branding Palette',  () => {
  it('matches brand colors', () => {
    expect(Palette).toMatchSnapshot()
    expect(Palette[500]).toBe('#00a4a3')
  })
})
