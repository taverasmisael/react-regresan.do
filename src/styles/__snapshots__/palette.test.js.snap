// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branding Palette matches brand colors 1`] = `
Object {
  "100": "#b3e4e3",
  "200": "#80d2d1",
  "300": "#4dbfbf",
  "400": "#26b2b1",
  "50": "#e0f4f4",
  "500": "#00a4a3",
  "600": "#009c9b",
  "700": "#009291",
  "800": "#008988",
  "900": "#007877",
  "A100": "#a6fffe",
  "A200": "#73fffe",
  "A400": "#40fffd",
  "A700": "#26fffd",
  "error": "#FF5252",
}
`;
